log/format.go

```diff
@@ -117,19 +119,19 @@ func TerminalFormat(usecolor bool) Format {
 				align = len(location)
 				atomic.StoreUint32(&locationLength, uint32(align))
 			}
-			padding := strings.Repeat(" ", align-len(location))
+			// padding := strings.Repeat(" ", align-len(location))
 
 			// Assemble and print the log heading
 			if color > 0 {
-				fmt.Fprintf(b, "\x1b[%dm%s\x1b[0m[%s|%s]%s %s ", color, lvl, r.Time.Format(termTimeFormat), location, padding, r.Msg)
+				fmt.Fprintf(b, "\x1b[%dm%s\x1b[0m %s %f [%s] %s", color, lvl, r.Time.Format(timeFormat), unixTime, location, r.Msg)
```

Comment:
what's the difference between `termTimeFormat` and `timeFormat`?

Reply:
Out of curiosity, what does the `\x1b[0m` portion of the format string do?

Reply:
those are hardcoded formats, defined at the top of this file. timeFormat := "12-20|23:52:11".
`\x1b[0m` is for color format. https://github.com/shiena/ansicolor/blob/master/README.md
0m sets all attributes off.